README.rst

```diff
@@ -202,7 +251,24 @@ Updates the version and date of the most recent changelog entry.
     Arguments:
       TAG       Version number to set on the entry.
 
+Config file
```

Comment:
Can you omit this change from this PR? I think this needs more discussion; I'm not convinced that chag needs a config file. If we really did need configuration, then it should live in the .git/config file. But even with that, I'm not sure it's needed.